documentation/prometheus-mixin/dashboards.libsonnet

```diff
@@ -0,0 +1,150 @@
+local g = import 'grafana-builder/grafana.libsonnet';
+
+{
+  dashboards+: {
+    'prometheus.json':
+      g.dashboard('Prometheus')
+      .addMultiTemplate('job', 'prometheus_build_info', 'job')
+      .addMultiTemplate('instance', 'prometheus_build_info', 'instance')
+      # Prometheus is quite commonly configured with honor_labels set to true;
```

Comment:
Huh? The Prometheus server is not something to use honor_labels true with.

Pgw, cloudwatch, statsd, graphite and influxdb exporters are the ones I know of.

Reply:
Agreed.

Reply:
Yeah, weird. I'll remove it.